client/client.go

```diff
@@ -398,9 +398,11 @@ func (c *NgsiV2Client) RetrieveEntity(id string, options ...RetrieveEntityParamF
 	if resp.StatusCode == http.StatusConflict {
 		return nil, fmt.Errorf("Conflict (id non-unique?).\nResponse body: %s", string(bodyBytes))
 	}
+
 	if resp.StatusCode != http.StatusOK {
 		return nil, fmt.Errorf("Unexpected status code: '%d'\nResponse body: %s", resp.StatusCode, string(bodyBytes))
 	} else {
+		fmt.Println(string(bodyBytes))
```

Comment:
Please remove the `Println`, it will mess the output of applications using the library and break their logging